util/include/json_output.hpp

```diff
@@ -78,4 +80,56 @@ inline std::pair<std::string, std::string> toPair(const std::string &key, const
   return std::make_pair(key, value);
 }
 
+// Build a JSON object.  Supports nested objects.
+class BuildJson {
+ public:
+  inline BuildJson() {}
```

Comment:
nit:
`BuildJson() = default;

question:
why do you need to explciitly declare it, when you don't have any other costume ctor?
The compiler builds an implcit ctor anyway..

Reply:
I've always seen ctors which do nothing declared this way.  That said, I did some googling and your position on this is well-supported.